src/main/java/uk/gov/hmcts/reform/blobrouter/exceptions/EnvelopeNotFoundException.java

```diff
@@ -1,4 +1,11 @@
 package uk.gov.hmcts.reform.blobrouter.exceptions;
 
 public class EnvelopeNotFoundException extends RuntimeException {
+
+    public EnvelopeNotFoundException() {
+    }
+
+    public EnvelopeNotFoundException(String message) {
+        super(message);
+    }
```

Comment:
I thought it would be nice to provide some context.
Not changing old code throwing this exception, hence the empty constructor.